engine/engine/src/test/wscript

```diff
@@ -14,10 +14,11 @@ def build(bld):
     if bld.env['PLATFORM'] in ('arm64-android','armv7-android'):
         graphics_lib = 'GRAPHICS_NULL DMGLFW' # g_AndroidApp is currently in glfw
 
+    exported_symbols = ['ProfilerExt', 'GraphicsAdapterNull', 'ResourceTypeGameObject', 'ResourceTypeCollection', 'ResourceTypeScript', 'ResourceTypeLua', 'ResourceTypeAnim']
```

Comment:
Using explicit symbols to make sure the linker doesn't optimize them away.

This ofc also means that we can now easily strip away resource types/components we don't want in a build.